modules/doc/content/ncrc/ncrc_local_binary.md

```diff
@@ -42,144 +42,176 @@ optional arguments:
+
+##### Running Tests
+
+To run the application's tests, you first instruct that application to copy its test files into a usable directory, enter said directory, and then run the tests. As follows:
+
+```bash
+$> conda activate bison
+$ (bison) >
+$ (bison) > bison-opt --copy-inputs tests
+
+<Trimmed results. Output consists of tests being copied.>
+
+$ (bison) > cd bison/tests
+$ (bison) > bison-opt --run -j 5
+fipd_rodlet_mesh_generator.test ........................................................................... OK
+fill_gas_thermal_conductivity.legacy.He ................................................................... OK
+<trimmed>
+triso_pebble.3d_pebble_from_csv .............................................................. [min_cpus=5] OK
+triso_pebble.cylinder_2d_rz ............................................................................... OK
+triso_pebble.cylinder_3d .................................................................................. OK
+triso_pebble.sphere_2d_rz ................................................................................. OK
+triso_pebble.sphere_3d .................................................................................... OK
+triso_pebble.plate ........................................................................................ OK
+--------------------------------------------------------------------------------------------------------------
+Ran 1606 tests in 274.6 seconds. Average test time 0.7 seconds, maximum test time 20.7 seconds.
+1606 passed, 22 skipped, 0 pending, 0 failed
+```
+
+##### Some Applications contain more than tests
+
+Some applications have different inputs that can be copied and executed. Each application can use whatever naming convention
+they wish, so listing all possibilities is impossible. However, any runnable input directory will have a file called `testroot`
+residing in it. The following *crazy* command will return those possibilities (using Bison as an example):
+
+```bash
+$> conda activate bison
+$ (bison) > find $CONDA_PREFIX/$NCRC_APP/share/$NCRC_APP -maxdepth 2 -type f -name 'testroot' \
+| awk 'BEGIN { FS = "/" } ; { print $(NF-1) }'
+workshop
+tools
+tests
+assessment
+examples
+benchmarks
+```
+
+##### Run a specific input file
+
+Unquestionably the real reason you installed said NCRC application, you can run your own input files for solving in the following way:
+
+```bash
+$> conda activate bison
+$ (bison) >
+$ (bison) > cd ~/path/to/my/input/file
+$ (bison) > bison-opt -i my_input.i
+# Using MPI
+$ (bison) > mpiexec -n 8 bison-opt -i my_input.i
+```
+
+##### View Documentation
+
+Documentation exists as part of the binary installation process. You can access documentation by instructing your favorite web browser to open the index.html file.
+We have made finding the location of this file via an environment variable, prefix with the NCRC application name:
```

Comment:
```suggestion
We have made finding the location of this file easier via an environment variable, prefixed with the NCRC application name:
```

Reply:
Its very much necessary in fact. Griffin for example my need/require a different set of libraries/versions of libMesh at the time the application was built.

Reply:
ok let's say it explicitly somewhere then? 

unless I missed it

Reply:
Well, actually I am wrong in how I originally defended the requirement. But they still should be separated. The ncrc assumes the application name is also the environment name.

Reply:
Yeah I'll make mention of this in the first alert on the page. In my defense, the ncrc script itself will actually warn/error when the user tries to use it while not in the base environment (except during upgrades).

Reply:
More content regarding separate environments for each app was added in subsequent commits. I think this is resolved.